commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -8,55 +8,84 @@ import squants.motion.AngularVelocity
 import squants.{Acceleration, Dimensionless, Length, Percent, Velocity}
 import GenericValue._
 
+trait UnicycleProperties {
+  val maxForwardVelocity: Velocity
+  val maxTurnVelocity: AngularVelocity
+
+  val forwardControlGains: PIDFProperUnitsConfig[Velocity, Acceleration, Length, Dimensionless]
+  val turnControlGains: PIDFConfig[AngularVelocity,
+                                   GenericValue[AngularVelocity],
```

Comment:
In Squants, AngularVelocity doesn't have a derivative or integral, but to do PID we want units that represent these values. So we have GenericValue (which wraps another unit), which can be integrated to a GenericIntegral or derived to a GenericDerivative.